roles/network_plugin/flannel/tasks/main.yml

```diff
@@ -6,8 +6,15 @@
     src: "{{item.file}}.j2"
     dest: "{{kube_config_dir}}/{{item.file}}"
   with_items:
-    - {name: flannel, file: cni-flannel-rbac.yml, type: sa}
-    - {name: kube-flannel, file: cni-flannel.yml, type: ds}
+    - { name: flannel, file: cni-flannel-rbac.yml, type: sa }
```

Comment:
We don't need to introduce a space between `{` and `name`?

Reply:
Fixed